README.md

```diff
@@ -2,132 +2,96 @@
-Fig. 2).
+will smile upon you as warmly as it did [the early hires](#history-and-significance) of
+Facebook. But a thorough overview can help you be informed when
+[discussing with professionals](#seeking-professional-advice) who may help you, make better
+decisions, avoid some of the most [common](#tax-dangers) and [costly](#the-amt-trap) mistakes.
 
 ### Why this Guide?
 
-The authors of this Guide created it to help everyone get a fair shake, understand their
-decisions, and save time and effort.
 A significant number of experts, editors, and authors have worked to assemble and review
-it.
-
-You shouldn’t need a law degree or MBA to understand how you are paid.
-You shouldn’t have to throw blind trust at a lawyer or tax professional.
-And you shouldn’t have to fail before you succeed.
+this Guide. Each of us are motivated by a single purpose:
+To help the reader navigate a confusing topic.
+The Holloway Guide is based on an earlier edition that has received significant discussion
+and feedback [on Hacker News](https://news.ycombinator.com/item?id=10880726) and
+[on GitHub](https://github.com/jlevy/og-equity-compensation). We thank
+[everyone who has helped](#credits) write, review, edit, and improve this work.
 
 While [a lot of information](#further-reading) on this topic is just a Google search away,
 the best resources are scattered about.
 Many blogs, books, and articles focus only on a narrow component of this topic, are out of
 date, or are on sites filled with ads or marketing copy.
-In contrast, the goals of this Guide are to be **reliable** (written and reviewed by
-credible experts, and not written to promote any company or product), **practical**
-(includes concrete details, advice, and pitfalls to avoid), **clear** (brief yet precise),
-and **thoughtful** (provides context and multiple expert perspectives, including divergent
-opinion on controversial topics).
-
-Finally, this Guide is **living**: We continue to update this material, and
-[welcome your contributions](#please-help)! This work benefits from your suggested
-improvements, requests for clarification or further information, and any and all
-questions.
-
-The Holloway edition of this Guide is based on an earlier edition that received
-significant discussion and feedback
-[on Hacker News](https://news.ycombinator.com/item?id=10880726) and
-[on GitHub](https://github.com/jlevy/og-equity-compensation). We thank
-[everyone who has helped](#credits) write, review, edit, and improve this work.
 
-### Should you use this Guide?
+In contrast, the goals of this Guide are to be **practical** (including lots of concrete
+details, advice, and pitfalls to avoid), **clear** (brief yet precise), and **thoughtful**
+(with context and multiple expert perspectives, including divergent opinion on
+controversial topics).
+
+Perhaps most importantly, Holloway Guides are **imperfect but improving**: It will continue
```

Comment:
I struggle with the word "imperfect." "Holloway Guides are **improving**" is a positive framing of the enterprise, without the fear prompted by the word "imperfect." "Improving" _implies_ that it is imperfect, so I think it's enough on its own. But the word itself, "imperfect," could build mistrust in the material that _is_ present (not what we want!), while "improving" just lets everyone know that there is material that we're missing and hoping to add, maybe even with your help.